README.md

```diff
@@ -1,4 +1,52 @@
-# ai-powered-search
-Source code and executable examples for the book _AI-Powered Search_ (Manning Publications)
+# AI-Powered Search
 
-Instructions for use are located in Appendix A of the book.
+Code examples for the book [_AI-Powered Search_](http://aipowerersearch.com) by [Trey Grainger](https://www.linkedin.com/in/treygrainger/), [Doug Turnbull](https://www.linkedin.com/in/softwaredoug/), and [Max Irwin](https://www.linkedin.com/in/maxirwin/). Published by [Manning Publications](http://manning.com).
```

Comment:
Change Manning URL to `https://www.manning.com/`

Reply:
Fixed. Also went through all notebooks and added titles and made hyperlinking between notebooks consistent (i.e. "Up next..." at end of notebooks, notebooks naming conventions, etc.).